### PR TITLE
DEV-2033: Collect only cpu temp metrics

### PR DESCRIPTION
**Pull request by jonhenrik13**
Second attempt at the temperature metrics. This tries to determine cpu temp only.

### CODE REVIEW

File: app/metrics/temperature_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		if len(files) == 0 {
+			// No temperature files found, skip test. CI uses vms and they don't have temperature files.
+			t.Skip("no temperature files found")
+		}
+	}
+
+	metrics, err := CollectTemperature()
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
```
Comment (piotrbulinski) [inline app/metrics/temperature_test.go:37]: It fails on my machine with:
```
=== RUN   TestCollectTemperature
    temperature_test.go:37: unexpected error: no temperature metrics found: no thermal zone metrics found
--- FAIL: TestCollectTemperature (0.00s)
```

I think the temperature metric collections should only produce an error when something unexpected happen (file exists but can't be opened due to permissions, etc...). But lack of thermal zones shouldn't cause an error, as not every device will have it.

**Review by piotrbulinski — CHANGES_REQUESTED** [review]

File: app/metrics/temperature_test.go (hunk 1 above)
Comment (piotrbulinski) [reply app/metrics/temperature_test.go]: What I meant by my comment, was that the `CollectTemperature()` function shouldn't produce an error when there are no thermal zones, as lack of thermal zones can be normal for some devices. We don't want an error message in the agent logs when this happens.

Comment (jonhenrik13) [reply app/metrics/temperature_test.go]: Skipping test on error

Comment (piotrbulinski) [reply app/metrics/temperature_test.go]: @jonhenrik13 I'm not sure if you noticed my recent comment. We shouldn't produce an error when no temperature metrics are available.

Comment (jonhenrik13) [reply app/metrics/temperature_test.go]: Not returning error, log in debug level

**Review by piotrbulinski — APPROVED** [review]

**Pull request merged**